### ISSUE TITLE
Use of `execute_process()` causes libc tests files to be generated every build

### ISSUE THREAD

**Issue by andschwa** [Microsoft/openenclave#1249]
This breaks incremental compilation. I thought it might be an issue in #782, but it wasn't a blocker. Now it's a bug.

/cc @mikbras

**Comment by andschwa** [comment]
```
...
Created /home/andschwa/openenclave/build/ninja/tests/libc/enc/3rdparty/musl/libc-test/src/functional/random.c
Created /home/andschwa/openenclave/build/ninja/tests/libc/enc/3rdparty/musl/libc-test/src/functional/time.c
Created /home/andschwa/openenclave/build/ninja/tests/libc/enc/tests.c
```

I was afraid of this 😆

**Comment by andschwa** [comment]
This was resolved via #1265.
